### ISSUE TITLE
Feature: Add separate option for disabling only startup notification

### ISSUE THREAD

**Issue by TheRedScreen64** [zocker-160/SyncThingy#27]
Currently I can only see an option for disabling all notifications. If the startup notification is the only one ment with that (as I can guess from #5 ), then a better labelling would be better for users to understand, and if not, a separate option for disabling only the startup notification would be great.

**Comment by zocker-160** [comment]
Currently that setting does actually only disable the startup and stop notifications. Error notifications will always be sent regardless.

So calling it `disable start/stop notifications` would probably be better?

I am not willing to add an option to disable all notifications, because that is something the user should be doing in the global system settings.

**Comment by TheRedScreen64** [comment]
Yes, I think that would make it clearer. It was confusing, at least for me, because I didn't want to disable all notifications.

**Comment by zocker-160** [comment]
ok yeah I will update the label, thank you for bringing this up.
